### DISCUSSION TITLE
ASUS ROG STRIX G15 2021

### DISCUSSION THREAD

**Discussion by Clyde911** [seerge/g-helper#462] [General]
Hello, i have just installed this cool tool to debloat my streaming laptop from Asus AC. I noticed there is no more dGPU power limit regulation when i set on turbo, and i dont even see the slider on the FANS POWER section, only core clock and memory cock offset; even on MSI i cannot more adjust the power limit manually. Im afraid it was a AC only feature, or is it possible to restore it with some ghelper updates?

On unigine i can still achieve a 7200 score on HIGH with just the offset +100 core +120 memory (instead of 7000 with stock), but the power limit in the AC Turbo mode achieved a 7800 score.

Thank you for your time!
Clyde

**Comment by seerge** [comment]
@Clyde911 hello,

>  I noticed there is no more dGPU power limit regulation when i set on turbo,

What do you mean under that ? Where did you had power limit before ?

**Reply by Clyde911** [reply]
> @Clyde911 hello,
> 
> > I noticed there is no more dGPU power limit regulation when i set on turbo,
> 
> What do you mean under that ? Where did you had power limit before ?

Oh sorry youre right, i never had it on g helper. It was here on AC and in the Turbo profile as a Rog Dynamic boost feature i think. Also i was able to regulate the Power limit in msi afterburner, now is it locked. 

The Turbo profile was giving +20w to the dgpu from what i had tested.

My question borns from the photo in the g helper main page here, where i see the Rog zephyrous with dgpu power and temps limits control, do you think there is a way to restore this feature also with g helper ? I really enjoy the debloat from AC, thank you for that man!

**Reply by seerge** [reply]
@Clyde911 g-helper shows dynamic boost slider for GPU only if bios reports it as available (apparently not in your case).

For experiment you can try to manually run in powershell as admin following command and see if it makes an effect: 

```
Invoke-CimMethod (Get-CimInstance -Namespace root/wmi -ClassName AsusAtkWmi_WMNB) -MethodName DEVS -Arguments @{Device_ID=0x1200C0; Control_status=20}
```

As for your Afterburner settings - they are controlled by nvidia driver / vbios i think ;) G-helper has nothing to do with that at  all.

You might check this topic tho https://www.reddit.com/r/GamingLaptops/comments/zs805q/bypassing_dynamic_boost_20_for_30_gen_laptops/

**Reply by Clyde911** [reply]
Thank you for your answers ! Apparently my laptop should support Dynamic boost as the specs sheet says 

"NVIDIA® GeForce RTX™ 3050 Ti Laptop GPU
 With ROG Boost up to 1795MHz at 80W (95W with Dynamic Boost)
 4GB GDDR6"

So it should appear the slider, i will try that command you posted. I see that ghelper is just Reading bios settings but im just wondering why im not getting either the slider.

Actually my max voltage now is 60w and im not able anymore to move the Power limit. I Remember to have reached the 80w with AC Turbo mode or also manually with msi afterburner slider. Dunno if is about driver, im using the OEM one GRD 516.40 i will try the last one from Nvidia website to see if something changes.

Im not in the mood to reinstall all the AC bloatware because im getting a more responsive PC now without it. Maybe i will just try to reinstall it for a while to do some extra test

Thank you anyway, i will post if i get some news!

**Reply by seerge** [reply]
I believe you, that it supports 80w wattage (not voltage). But app doesn't lower that anyhow :) It's either limited by driver (then check reddit link from previous post on how to "remove" limits) or by bios (then powershell command should work as this is the only known and discovered by me way to adjust that)

**Reply by Clyde911** [reply]
Here what i did this morning: i updated the Nvidia Driver with the latest one on the site, and in the Nvidia panel i seen the Dynamic boost 2.0 active which wasnt before.

So i did a benchmark and i got
75w on silent/balanced profiles (60w base+15w Dynamic boost)
95w on turno profile (60w+20w by vbios profile i think and 15w from DB)

Now, i did a clear reinstall of the OEM drivers and this time the DB is here, not in the ghelper but in the nvidia panel and it works as said before. Anyway i dont know how much is worth as the temperatures go much higher. So maybe i will keep the latest driver to use the Dynamic boost bypass you mentioned before to have more control on the GPU and leave that poor 4800h CPU alone, as i mentioned before i dont use this pc much for gaming but just for vmix live works so i need stability and low latency from both cpu/gpu, not horsepower

**Reply by seerge** [reply]
@Clyde911 ok, glad to hear it has worked out well:)

**Comment by Clyde911** [comment]
Hello seerge and again thank you for your great work!

So after months i finally realized what breaks the GPU wattage in my laptop (ROG G15 2021 G513IE - RYZEN 4800H RTX 3050TI 95W)
**It was because of the customized fan profile i tend to always set myself.**

How i find out:
Since i was stubborn to find why sometimes the gpu pushes 95w on turbo profile an sometimes only 75w i some days ago i reinstalled AC bloatware and turned off Ghelper for research purpose, cause id like to have actual control over my laptop.
The turbo profile was pushing 95w without problem always (es. passing from eco to dgpu mode, rebooting, unplugging then plugging the wall power, changing performance profile, etc), since i wanted more control on the noise i created a custom profile with custom fan curve and while benchmarking i saw only 75w. So i told to myself the 95w is baked into the Turbo profile.

Then i gave up again on AC and unistalled it, reactivated ghelper, all profiles was working fine until i customized the fan curve and in the exact moment i pressed APPLY FAN CURVE the GPU wattage falled from 95w to 75w! Unticking reverted immediately the situation to 95.

The laptop site specs says:
"NVIDIA® GeForce RTX™ 3050 Ti Laptop GPU 
With ROG Boost up to 1795MHz at 80W (95W with Dynamic Boost)
4GB GDDR6"

So i tested every profile with hwinfo and gpuz:
-Silent uses 75w: 55+20 of ROG boost
-Balanced uses 80w: 60+20 of ROG boost
-Turbo uses 95w: 60+20 ROG boost + 15w of Dynamic Boost 2.0 which is not tweakable but baked into the profile at BIOS level i suppose)
-Custom profile or customizing the fan curve of stock preset uses the same config as Silent profile.

To conclude i also managed to undervolt to a stable -10 my 4800h cpu, not a big deal about performance (just +100pt on cinebench R23) or full load temps, but surely it helps a little on normal loads.

Hope this material can helps someone else!

Bye!
Clyde

**Comment by Clyde911** [comment]
> Hello, i have just installed this cool tool to debloat my streaming laptop from Asus AC. I noticed there is no more dGPU power limit regulation when i set on turbo, and i dont even see the slider on the FANS POWER section, only core clock and memory cock offset; even on MSI i cannot more adjust the power limit manually. Im afraid it was a AC only feature, or is it possible to restore it with some ghelper updates?
> 
> On unigine i can still achieve a 7200 score on HIGH with just the offset +100 core +120 memory (instead of 7000 with stock), but the power limit in the AC Turbo mode achieved a 7800 score.
> 
> Thank you for your time! Clyde

Ah! and now i achieved 8169pts on the same benchmark, i can confirm a nice performance improvement on both CPU/GPU with ghelper instead of AC.
